build.gradle.kts

```diff
@@ -89,10 +89,12 @@ tasks.withType<DependencyUpdatesTask> {
 }
 
 project.tasks.withType<org.jetbrains.kotlin.gradle.tasks.UsesKotlinJavaToolchain>().configureEach {
-    kotlinJavaToolchain.jdk.use(
-        "/usr/lib/jvm/java-16-openjdk-amd64",
-        JavaVersion.VERSION_16
-    )
+    val service = project.extensions.getByType<JavaToolchainService>()
+    val customLauncher = service.launcherFor {
+        this.languageVersion.set(JavaLanguageVersion.of(JavaVersion.VERSION_16.majorVersion))
+    }
+
+    this.kotlinJavaToolchain.toolchain.use(customLauncher)
```

Comment:
This change removes the requirement that the jdk lives at `/usr/lib/jvm/java-16-openjdk-amd64`